co-simulation tool/bridge/carla_mosaic_bridge.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2020 Computer Vision Center (CVC) at the Universitat Autonoma de
+# Barcelona (UAB).
+#
+# This work is licensed under the terms of the MIT license.
+# For a copy, see <https://opensource.org/licenses/MIT>.
+
+# ==================================================================================================
+# -- imports ---------------------------------------------------------------------------------------
+# ==================================================================================================
+
+import argparse
+import logging
+import shutil
+import tempfile
+
+# ==================================================================================================
+# -- find carla module -----------------------------------------------------------------------------
+# ==================================================================================================
+
+import glob
+import os
+import sys
+
+try:
+    sys.path.append(
+        glob.glob('PythonAPI/carla/dist/carla-*%d.%d-%s.egg' %
+                  (sys.version_info.major, sys.version_info.minor,
+                   'win-amd64' if os.name == 'nt' else 'linux-x86_64'))[0])
+except IndexError:
+    pass
```

Comment:
Is that possible to handle this exception? either showing message in the terminal or saving in the log file

Reply:
The exception has been handled by showing message in the terminal.